src/sql/magic_cmd.py

```diff
@@ -38,12 +34,38 @@ class SqlCmdMagic(Magics, Configurable):
     @line_magic("sqlcmd")
     @magic_arguments()
     @argument("line", default="", type=str, help="Command name")
-    def execute(self, line="", cell=""):
+    def _validate_inputs(self, line=""):
```

Comment:
Since we are validating the inputs (i.e: line), let's replace `line = ""` with  `line`.

Also, let's rename `_validate_inputs` to `_validate_execute_inputs` to make it clear.